app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -872,6 +875,14 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Session
         }
     }
 
+    /**
+     * Set the activity normal/private theme to match the current session.
+     */
+    private fun updateThemeForSession(session: Session) {
```

Comment:
We can probably remove the fun in HomeActivity then if you moved it here? I think these are the only two calls to it.